src/components/Combobox/Combobox.jsx

```diff
@@ -27,6 +27,7 @@ const Combobox = forwardRef(
       disabled,
       options,
       categories,
+      categoriesDivider,
```

Comment:
please change the prop name to "withCategoriesDivider"

Reply:
V